go-controller/pkg/ovn/namespace.go

```diff
@@ -53,6 +53,14 @@ func (oc *Controller) addPodToNamespace(ns string, portInfo *lpInfo) error {
 	}
 	defer nsInfo.Unlock()
 
+	if nsInfo.addressSet == nil {
+		nsInfo.addressSet, err = oc.createNamespaceAddrSetAllPods(ns)
```

Comment:
Maybe put this in `waitForNamespaceLocked` instead? (That would handle both this case and the `addEgressFirewall` case above.)

Reply:
I think leave it since we do not want to create for EgressFirewall, and it is more explicit being here.